DataFormats/simulation/src/SimulationDataLinkDef.h

```diff
@@ -44,7 +44,7 @@
 #pragma link C++ class o2::BasicXYZQHit < double, double, int> + ;
 #pragma link C++ struct o2::dataformats::MCTruthHeaderElement + ;
 #pragma link C++ class o2::dataformats::MCTruthContainer < long> + ;
-#pragma link C++ class o2::dataformats::MCTruthContainer < o2::MCCompLabel> + ;
+#pragma link C++ class o2::dataformats::MCTruthContainer < o2::MCCompLabel > -;
```

Comment:
Never saw this before, what is the special meaning of `-` ?

Reply:
with `+` , rootcling will add the default implementation for the `Streamer` function, `-` means this is skipped and one can add a custom implementation.